visualizations/frontend/ResetAll.vue

```diff
@@ -0,0 +1,19 @@
+<template>
```

Comment:
I wasn't sure where the right place to store this component was. I put it in `visualizations/frontend/ResetAll.vue`, but would be open to suggestions.